### ISSUE TITLE
loss.detach()

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#22]
Don't store loss with grad_fn in losses history, but only the loss tensor without the grad function.
